### DISCUSSION TITLE
connecting artisan to 2 wireless vint hubs

### DISCUSSION THREAD

**Discussion by ivanroaster** [artisan-roaster-scope/artisan#1138] [General]
hello forum, what is the way to connect artisan to 2 wireless vint hubs? is this possible?
I can connect fine with the remote checkbox to the ip of one of the vint hubs but not sure if I can access two wireless vint hubs at the same time from Artisan (to have access to 6+6=12 vint ports). I could minimize my setup to use only one wireless vint hub if two are not supported.
I tried running a network server but I do not see the wireless hubs as local devices in my network server...
Sorry if this has been resolved in the past, I didn't find the answer.
thanks for the help!

**Comment by MAKOMO** [comment]
Artisan harvests all networked Phidgets reachable on the network, connected to a wireless hub or a computer running an active Phidgets network server, if no host or password is set in the Artisan Phidgets control panel and the Phidgets server are discoverable via the ZeroConf protocol.

For the lack of a second HUB5000 I started a Phidget Network Server running on a second computer on the same network connected to a Phidget 1048 with one TC connected to port 0. A TMP1100 is connected to a HUB5000. Both modules are successfully harvested by Artisan.

<img width="1170" alt="Screenshot 2023-04-18 at 12 18 43" src="https://user-images.githubusercontent.com/13909011/232748538-256d388a-bcea-4e7d-bf9d-d57212742f7f.png">

<img width="1055" alt="Screenshot 2023-04-18 at 12 15 43" src="https://user-images.githubusercontent.com/13909011/232748534-8038d31e-b438-4fe6-9f55-74c700cee2f7.png">
